Vagrantfile

```diff
@@ -11,29 +11,30 @@ Vagrant.configure("2") do |config|
 
   config.vm.provision "shell", inline: <<-SHELL
     apt-get update
-    apt-get install -y --no-install-recommends wget curl jq shellcheck bsdmainutils psmisc
+    apt-get install -y --no-install-recommends git wget curl jq \
+        make shellcheck bsdmainutils psmisc
 
     wget -qO- https://get.docker.com/ | sh
     usermod -a -G docker vagrant
     apt-get autoremove -y
 
-    apt-get install -y --no-install-recommends git
```

Comment:
i'd prefer this line to stay here because it indicates that go requires it! maybe we should add a comment though

Reply:
I moved it up higher for consistency. But sure, I can put it back.